src/index.html

```diff
@@ -11,6 +11,19 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="search__form" data-qa="big">
+      <input
+              class="search__form__big"
+              data-qa="keypress"
+              placeholder="Try &quot;Los Angeles&quot;"
+      >
```

Comment:
Let's fix the code style. Also add name attribute for input
```suggestion
      <input
        class="search__form__big"
        data-qa="keypress"
        placeholder="Try &quot;Los Angeles&quot;"
      >
```